### PR TITLE
Syncing styling

### PR DESCRIPTION
**Pull request by ayumi**
also show device name

Auditors: @diracdeltas

screen shot 2017-02-09 at 18 35 13
screen shot 2017-02-09 at 18 35 35
screen shot 2017-02-09 at 18 35 41
screen shot 2017-02-09 at 18 35 52

### CODE REVIEW

**Comment by diracdeltas** [comment]
[minor] the large right-side padding is a little weird
screen shot 2017-02-10 at 4 47 26 am

some of the syncing tests are unstoked https://travis-ci.org/brave/browser-laptop/builds/200218416#L4843

File: less/about/preferences.less [hunk 1]
```diff
@@ -1561,56 +1557,74 @@ table.sortableTable {
...
+      justify-content: flex-start;
+    }
   }
-  textarea {
-    width: 80%;
-    height: 100px;
-    font-size: 18px;
-    font-family: monospace;
+  .browserButton + .browserButton {
+    margin-left: 0.75em;
```
Comment (luixxiul) [inline less/about/preferences.less]: I think this should be covered with https://github.com/brave/browser-laptop/blob/2aa39bb1073fe8bb40d9fc32d18fdd2ea9cee1d5/less/button.less#L52

File: less/forms.less [hunk 2]
```diff
@@ -764,3 +764,19 @@ select {
     }
   }
 }
+
+// From commonStyles.js
```
Comment (luixxiul) [inline less/forms.less]: I am not quite if it should be recovered: https://github.com/brave/browser-laptop/commit/4fb5d5ae8202b5afffff4e3ce106a64d81fbfdb4 CC @bsclifton for a comment

Comment (luixxiul) [inline less/forms.less]: If needed temporarily, `TODO: ...`  should be added for a future task

File: app/renderer/components/preferences/syncTab.js [hunk 3]
```diff
@@ -56,7 +65,7 @@ class SyncTab extends ImmutableComponent {
     }
     return this.props.syncQRVisible
       ? <div>
-        <div><Button l10nId='syncHideQR' className='whiteButton syncToggleButton' onClick={this.props.hideQR} /></div>
+        <div><Button l10nId='syncHideQR' className='whiteButton wideButton syncToggleButton' onClick={this.props.hideQR} /></div>
```
Comment (luixxiul) [inline app/renderer/components/preferences/syncTab.js]: Do we need this one? the min-width of the buttons has been set globally with https://github.com/brave/browser-laptop/pull/6387

File: app/renderer/components/preferences/syncTab.js [hunk 4]
```diff
@@ -76,7 +85,7 @@ class SyncTab extends ImmutableComponent {
     ]
     return this.props.syncPassphraseVisible
       ? <div>
-        <Button l10nId='syncHidePassphrase' className='whiteButton syncToggleButton' onClick={this.props.hidePassphrase} />
+        <Button l10nId='syncHidePassphrase' className='whiteButton wideButton syncToggleButton' onClick={this.props.hidePassphrase} />
```
Comment (luixxiul) [inline app/renderer/components/preferences/syncTab.js]: Ditto

**Comment by luixxiul** [comment]
@ayumi I created an new PR #7164 to propose an improvement. Would you please take a look at it?

File: app/renderer/components/preferences/syncTab.js (hunk 3 above)
Comment (ayumi) [inline app/renderer/components/preferences/syncTab.js]: I applied this class as a hack to make the "button pressed / secret visible" state have a larger button. wideButton might be inappropriate so I can explicitly set a width for these buttons.

File: less/about/preferences.less (hunk 1 above)
Comment (ayumi) [inline less/about/preferences.less]: 5px was insufficient :)

Comment (luixxiul) [inline less/about/preferences.less]: Surely!

**Comment by ayumi** [comment]
@diracdeltas ran [tests again](https://travis-ci.org/brave/browser-laptop/builds/200218412); the test failures look to be intermittent and I'll try to address it in a separate PR.

Adjusted switch/name spacing based on feedback:
screen shot 2017-02-10 at 09 59 12

**Pull request merged**